RecoDataProducts/inc/CosmicTrackSeed.hh

```diff
@@ -0,0 +1,50 @@
+#ifndef RecoDataProducts_CosmicTrackSeed_hh
+#define RecoDataProducts_CosmicTrackSeed_hh
+//
+// S. Middleton - seed class for striaght track fit
+// Feb 2019
+
+// Mu2e includes
+#include "BTrk/TrkBase/TrkT0.hh"
+#include "TrackerGeom/inc/Tracker.hh"
+#include "RecoDataProducts/inc/ComboHit.hh"
+#include "RecoDataProducts/inc/TrkStrawHitSeed.hh"
+#include "RecoDataProducts/inc/CosmicTrack.hh"
+#include "RecoDataProducts/inc/TimeCluster.hh"
+#include "RecoDataProducts/inc/CosmicTrkFitFlag.hh"
+#include "canvas/Persistency/Common/Ptr.h"
+#include <vector>
+
+namespace mu2e {
+  
+  class TimeCluster;
+
+  struct CosmicTrackSeed {
+
+    TrkT0 const& t0() const { return _t0; }//from the Time cluster
+    const std::vector<StrawHitIndex>& _shits       () const { return _strawHitIdxs; }
+    
+    std::vector<TrkStrawHitSeed> const& trkstrawhits() const { return _trkstrawhits;}
+    ComboHitCollection const& hits() const { return _panel_hits; }
+    CosmicTrack const& track() const { return _track; }
+    CosmicTrkFitFlag const& status() const { return _status; }
+    art::Ptr<TimeCluster> const& timeCluster() const { return _timeCluster; }
+    
+    TrkT0	             _t0;	      // t0 for this track
+    ComboHitCollection       _panel_hits;	      // hits for track (panel hits)
+    ComboHitCollection       _straw_chits;    // get the straw level hits and store here (need to find panel hits first)
```

Comment:
Is there a reason you are deep copying the ComboHits, and not just keeping indices as in TimeCluster?

Reply:
investigating....I think there is a reason....